web/src/test/specs.md

```
@@ -0,0 +1,147 @@
+# BDD Test Specifications
+
+## Feature: User Authentication
+
+### Scenario: User can sign in with Google OAuth
+**Given** the user is on the login page  
+**When** the user clicks the "Sign in with Google" button  
+**Then** the user should be redirected to Google's OAuth flow  
+**And** after successful authentication, the user should be redirected back to the app  
+**And** the user should be logged in  
+
+### Scenario: User can sign-out
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)